### PR TITLE
LevelDB-backed dwn via expo-level

### PR DESCRIPTION
**Pull request by amika-sq**
`expo-level` ([link](https://www.npmjs.com/package/expo-level)) is an `abstract-level` adapter for `react-native-leveldb`. 

Because it provides a complete `abstract-level` interface, we can use it directly when initializing `MessageStoreLevel`, `DataStoreLevel`, and `EventLogLevel`.

Addresses issue https://github.com/TBD54566975/web5-wallet/issues/47

### CODE REVIEW

File: src/features/dwn/dwn-service.ts [hunk 1]
```diff
@@ -1,43 +1,42 @@
...
   EventLogSql,
   MessageStoreSql,
   SqliteDialect,
 } from "@tbd54566975/dwn-sql-store";
 import { SqliteDatabase } from "./sqlite-config";
+import {
+  MessageStoreLevel,
+  DataStoreLevel,
+  EventLogLevel,
+} from "@tbd54566975/dwn-sdk-js/dist/esm/src/index-stores";
```
Comment (amika-sq) [inline src/features/dwn/dwn-service.ts:20]: `MessageStoreLevel`, `DataStoreLevel`, and `EventLogLevel` are no longer exported by default in the `dwn-sdk-js` package. Instead, they are exposed via the `stores` export [here](https://github.com/TBD54566975/dwn-sdk-js/blob/main/package.json#L57). 

React Native currently doesn't have named export support (but it is coming soon), so I'm just manually referencing that file here. Open to other options if there's a better way to do this for the interim!

Comment (shamilovtim) [reply src/features/dwn/dwn-service.ts:20]: We gotta get these exports added back. There was no great reason to remove them, as we needed all of our polyfills and bundler configs no matter what.

**Comment by shamilovtim** [comment]
lets inline expo-level into the `assets` folder:
`yarn add file:assets/expo-level/`

File: assets/expo-level/package.json [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      "optional": true
+    }
+  },
+  "devDependencies": {
+    "@commitlint/config-conventional": "^11.0.0",
+    "@release-it/conventional-changelog": "^2.0.0",
+    "@types/react": "17.0.2",
+    "@types/react-native": "0.68.2",
+    "commitlint": "^11.0.0",
+    "react-native-builder-bob": "^0.18.0",
```
Comment (shamilovtim) [inline assets/expo-level/package.json:58]: interesting that `react-native-builder-bob` was used for this as the project is pure JS. we can probably edit in the future to remove all of this boilerplate if we fork

**Review by shamilovtim — APPROVED** [review]

**Pull request merged**